src/current/v23.1/row-level-ttl.md

```diff
@@ -548,6 +548,8 @@ Row-level TTL interacts with [changefeeds]({% link {{ page.version.version }}/cr
 
 - When expired rows are deleted, a [changefeed delete message]({% link {{ page.version.version }}/changefeed-messages.md %}#delete-messages) is emitted.
 
+Refer to [Change Data Capture Queries]({% link {{ page.version.version }}/cdc-queries.md %}#reference-ttl-in-a-cdc-query) for guidance on how to filter changefeed messages to emit row-level TTL deletes only.
```

Comment:
because my brain has been ruined by the Microsoft Manual of Style for Technical Publications (3rd ed., p. 33), I suggest that the clauses be reversed here, e.g. "for guidance on ..., see ..."